ci/docs.go

```diff
@@ -1,13 +1,21 @@
 package main
 
 import (
+	"context"
+	"fmt"
 	"os"
 	"os/exec"
+	"path/filepath"
+	"strings"
 
+	"dagger.io/dagger"
 	"github.com/urfave/cli/v2"
 )
 
-const techdocsImage = "squidfunk/mkdocs-material:9.1.4"
+const localTechdocsImage = "squidfunk/mkdocs-material:9.1.4"
```

Comment:
For local development it's easier to work with the default mkdocs-material image. The backstage one need to make some modifications to the mkdocs.yaml file when running which gets in the way when you just want to work with it.